packit_service/worker/helpers/testing_farm.py

```diff
@@ -1476,15 +1476,24 @@ def is_fmf_configured(project: GitProject, metadata: EventData) -> bool:
         ),
     )
     def _payload_custom(self, distro: str, compose: str) -> dict:
-        return {
-            "test": {
-                "tmt": {
-                    "url": self.project.get_pr(self.metadata.pr_id)
-                    .source_project.get_git_urls()
-                    .get("git"),
-                    "ref": self.metadata.commit_sha,
-                },
+        payload = self._get_tf_base_payload(distro, compose)
+        payload["test"] = {
+            "tmt": {
+                "url": self.project.get_pr(self.metadata.pr_id)
+                .source_project.get_git_urls()
+                .get("git"),
+                "ref": self.metadata.commit_sha,
             },
+        }
+        return payload
+
+    def _get_tf_base_payload(self, distro: str, compose: str) -> dict:
+        """
+        Common payload for all fedora-ci testing-farm jobs.
+
+        Does not contain the ``test`` field
```

Comment:
I think that

    `test`

should be enough here 🤔 why did you do double backticks?

Reply:
https://github.com/packit/contributing?tab=readme-ov-file#docstrings

Reply:
Good to know. Google-style is still rst based though, and similarly with numpy and such. Markdown docstrings are barely supported with myst-parser, and even then `sphinx-autodoc` is not supported. And IDEs are even worse on that support.